[RM Ch04-05] Consistency and Typo Corrections (#1756)
cntt-n/CNTT: doc/ref_model/chapters/chapter04.md

**a.b.c** (Ex. "e.pm.001")  
a = Scope <(e)xternal | (i)nternal | (t)hird_party_instrumentation>  

> b = Type <(cap)ability | (man)agmenet | (pm) performance | (man-pm)>  

Reviewer: ```suggestion
b = Type <(cap)ability | (man) managmenet | (pm) performance | (man-pm)>  
```

| Ref       | Cloud Infrastructure Capability           | Unit   | Definition/Notes                                            |
|-----------|-------------------------------------------|--------|-------------------------------------------------------------|
| e.cap.006 | CPU pinning                               | Yes/No | Indicates if Cloud Infrastructure supports CPU pinning      |

> | e.cap.007 | NUMA alignment                            | Yes/No | Indicates if Cloud Infrastructure supports NUMA             |

Reviewer: ```suggestion
| e.cap.007 | NUMA alignment                            | Yes/No | Indicates if Cloud Infrastructure supports NUMA alignment |
```

| e.cap.008 | IPSec Acceleration                        | Yes/No | IPSec Acceleration                                          |
| e.cap.009 | Crypto Acceleration                       | Yes/No | Crypto Acceleration                                         |
| e.cap.010 | Transcoding Acceleration                  | Yes/No | Transcoding Acceleration                                    |
| e.cap.011 | Programmable Acceleration                 | Yes/No | Programmable Acceleration                                   |

> | e.cap.012 | Enhanced Cache Management                 | Yes/No | If supported, L=Lean; E=Equal; X=eXpanded.  L and X cache policies require CPU pinning to be active. |

Reviewer: I think these should go into notes under the table.

Author: @CsatariGergely GSMA is asking that these be included within the text.

Reviewer: Okay

> | e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type.  |

Reviewer: ```suggestion
| e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As there is no alternate technology exists at the moment there is an exception provided for SR-IOV over PCI-PT in [rm.exc.001](appendix-a.md#a3-exception-list) |
```

> | e.cap.013 | SR-IOV over PCI-PT                        | Yes/No | Traditional SR-IOV. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type.  |

Reviewer: I think a reference to the [exception](https://github.com/cntt-n/CNTT/blob/master/doc/tech/policies.md#7131-sr-iov-over-pci-pt) is enough instead of re-explining the thing.

Author: @CsatariGergely The exception was listed as a note but GSMA requires that be in the text. Any reference would have to be content within the GSMA document

> | e.cap.014 | GPU/NPU                                   | Yes/No | Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type. |

Reviewer: Do we need an exception for these?

Reviewer: #1764 created for the missing exception.

> | e.cap.015 | SmartNIC                                  | Yes/No | Network Acceleration. SmartNICs that do not utilise PCI-PT are not subject to the CNTT principles, nor any related policies or prohibitions. |

Reviewer: ```suggestion
| e.cap.015 | SmartNIC                                  | Yes/No | Network Acceleration.  |
```

Author: @CsatariGergely The text is from a note that existed below the table and as per GSMA should be included within the text.

> | e.cap.016 | FPGA/other Acceleration H/W               | Yes/No | Non-specific hardware. These Capabilities generally require hardware-dependent drivers be injected into workloads, which is prohibited by CNTT principles. As such, use of these features shall be governed by the applicable CNTT policy. Please consult the RM Appendix for the usage policy relevant to any needed hardware Capability of this type. |

Reviewer: Do we need an exception for these?

Author: @CsatariGergely The exceptions would have to be in a section that is part of the rM for a self-contained GSMA document.

Reviewer: #1765 created to address the missing exception

| e.man.006 | Virtual resources inventory per tenant     | Yes/No | Capability to provide information related to allocated virtualised resources per tenant     |
| e.man.007 | Resources Monitoring                       | Yes/No | Capability to notify state changes of allocated resources                          |
| e.man.008 | Virtual resources Performance              | Yes/No | Capability to collect and expose performance information on virtualised resources allocated |
| e.man.009 | Virtual resources Fault information        | Yes/No | Capability to collect and notify fault information on virtualised resources                 |

<p align="center"><b>Table 4-10:</b> Cloud Infrastructure Management Performance Measurement Capabilities</p>

<a name="4.1.6.1"></a>
#### 4.1.6.1 Resources Management Measurements
**Table 4-11** shows resource management measurements of CIM as aligned with ETSI GS NFV TST-012 [3].

| Ref          | Cloud Infrastructure Management Measurement                   | Unit   | Definition/Notes |
|--------------|---------------------------------------------------------------|--------|------------------|

> | e.man-pm.001 | Time to create Virtual Compute resources for a given workload | Max ms | This is typically for a component: VNFC or CNFC |

Reviewer: We try to avoid the term VNFC (to avoid the confusion from the fact that VNFC can be a VM or a container, but can not be a Pod) and CNFC is a nonexisting thing. I would propose to use VM or Pod instead of them.

Author: @CsatariGergely added "(VM/conatiner)" after "Virtual Compute resources" and removed the comment from Notes. The intent was to use CNFC to represent a microservice as a CNF is a composition of a set of microservices.
